pallets/api/src/fungibles/mod.rs

```diff
@@ -193,15 +241,24 @@ pub mod pallet {
 		/// - `spender` - The account that is allowed to spend the tokens.
 		/// - `value` - The number of tokens to increase the allowance by.
 		#[pallet::call_index(6)]
-		#[pallet::weight(AssetsWeightInfoOf::<T>::approve_transfer())]
+		#[pallet::weight(AssetsWeightInfoOf::<T>::approve_transfer() + T::DbWeight::get().reads(1))]
```

Comment:
Missing proof size in this simple read addition? Sadly should have a benchmark.

Reply:
Please see commit `fix: weight increase allowance`. This is a simple solution and not 100% accurate however before I put more time into this I would like to get your view on the approach that has been taken for `decrease_allowance`. Basically, we are reusing the weight function of `approve` for `increase_` and `decrease_allowance`. This is for all three we have to read state (`allowance`) and either call `approve_transfer`, `cancel_approval`, or both. As I think this is fine for now since we never underestimate weight, this does deserve attention before going to mainnet.